lib/shared/widgets/apartment_details_sheat.dart

```diff
@@ -0,0 +1,188 @@
+        ),
+        SizedBox(
+          width: 336,
+          child: Text(
+            accomodation.shortDescription,
+            style: textTheme.bodyText1!.merge(
+              TextStyle(color: ThemeColors.teal800),
+            ),
+            textAlign: TextAlign.left,
+          ),
+        ),
+        const SizedBox(
+          width: 30,
+          height: 30,
+        ),
+        SizedBox(
+          width: 336,
+          child: Text(
+            accomodation.longDescription,
+            style: textTheme.bodyText1!.merge(
+              TextStyle(color: ThemeColors.teal800),
+            ),
+            textAlign: TextAlign.left,
+          ),
+        ),
+        const SizedBox(
+          width: 20,
+          height: 20,
+        ),
+        SizedBox(
+          width: 336,
+          child: Text(
+            "Property info:",
+            style: textTheme.bodyText1!.merge(
+              const TextStyle(color: Colors.black),
+            ),
+            textAlign: TextAlign.left,
+          ),
+        ),
+        const SizedBox(
+          width: 16,
+          height: 16,
+        ),
+        FittedBox(
+          fit: BoxFit.fitWidth,
+          child: Text(
+            '${accomodation.capacity} guests',
+            style: textTheme.bodyText2!.merge(
+              TextStyle(color: ThemeColors.teal800),
+            ),
+            textAlign: TextAlign.left,
+          ),
+        ),
+        const SizedBox(
+          width: 10,
+          height: 10,
+        ),
+        FittedBox(
+          fit: BoxFit.fitWidth,
+          child: Text(
+            accomodation.accommodationType,
+            style: textTheme.bodyText2!.merge(
+              TextStyle(color: ThemeColors.teal800),
+            ),
+            textAlign: TextAlign.left,
+          ),
+        ),
+        const SizedBox(
+          width: 10,
+          height: 10,
+        ),
+        FittedBox(
+          fit: BoxFit.fitWidth,
+          child: Text(
+            'EUR ${accomodation.price} per night',
+            style: textTheme.bodyText2!.merge(
+              TextStyle(color: ThemeColors.teal800),
+            ),
+            textAlign: TextAlign.left,
+          ),
+        ),
+        const SizedBox(
+          width: 10,
+          height: 10,
+        ),
+        FittedBox(
+          fit: BoxFit.fitWidth,
+          child: Text(
+            accomodation.location,
+            style: textTheme.bodyText2!.merge(
+              TextStyle(color: ThemeColors.teal800),
+            ),
+            textAlign: TextAlign.left,
+          ),
+        ),
+        const SizedBox(
+          width: 10,
+          height: 10,
+        ),
+        FittedBox(
+          fit: BoxFit.fitWidth,
+          child: Text(
+            accomodation.postalCode,
+            style: textTheme.bodyText2!.merge(
+              TextStyle(color: ThemeColors.teal800),
+            ),
+            textAlign: TextAlign.left,
+          ),
+        ),
+        Center(
```

Comment:
This button should be always visible and floating on bottom of the screen. You can remove it from here and add it on apartment_details_screen.dart. (Look at comment there)